scylla/src/transport/load_balancing/default.rs

```diff
@@ -720,8 +708,15 @@ impl DefaultPolicyBuilder {
     /// the default policy will only include local nodes in load balancing plans.
     /// Remote nodes will be excluded, even if they are alive and available
     /// to serve requests.
-    pub fn prefer_datacenter(mut self, datacenter_name: String) -> Self {
-        self.preferred_datacenter = Some(datacenter_name);
+    ///
+    /// When a preferred rack is set, the policy will first return replicas in the local rack
+    /// in the preferred datacenter, and then the other replicas in the datacenter.
+    pub fn prefer_datacenter_and_rack(
```

Comment:
That's a breaking change. I thought we wanted to include the LWT optimization in 0.8.2, but it looks like we will have to release 0.9.

Reply:
My idea was to merge LWT optimisation for 0.8.2 and the API change for 0.9/1.0.

Reply:
OK, we can consider this for a later, major release. I see that the commits were dropped from the PR, so I'm resolving the conversation.